sdk/core/azure-core/inc/azure/core/url.hpp

```diff
@@ -28,7 +28,7 @@ namespace Azure { namespace Core {
           auto separator = '?';
           for (const auto& q : encodedQueryParameters)
           {
-            queryStr += separator + q.first + '=' + q.second;
+            queryStr += separator + q.first + '=' + q.second; // LCOV_EXCL_LINE
             separator = '&';
```

Comment:
Why is line 31 not covered, but line 32 is? 🤔

Reply:
Because theoretically every string concatenation might throw, if system runs out of memory.